lib/spack/spack/main.py

```diff
@@ -25,6 +25,7 @@
 import spack.architecture
 import spack.config
 import spack.cmd
+import spack.environment as ev
```

Comment:
Let's stick with `ev` for now -- I like `env` better as a variable name, so I tried to avoid using it as a namespace.  This is kind of following in the `import numpy as np` and `import pandas as pd` tradition of 2-letter namespaces...